api/src/test/java/org/openmrs/util/DatabaseIT.java

```diff
@@ -55,40 +58,64 @@ public Liquibase getLiquibase(String filename) throws LiquibaseException, SQLExc
 		Database liquibaseConnection = DatabaseFactory.getInstance()
 		        .findCorrectDatabaseImplementation(new JdbcConnection(getConnection()));
 		
-		liquibaseConnection.setDatabaseChangeLogTableName("LIQUIBASECHANGELOG");
-		liquibaseConnection.setDatabaseChangeLogLockTableName("LIQUIBASECHANGELOGLOCK");
+		liquibaseConnection.setDatabaseChangeLogTableName("liquibasechangelog");
+		liquibaseConnection.setDatabaseChangeLogLockTableName("liquibasechangeloglock");
 		
 		return new Liquibase(filename, new ClassLoaderResourceAccessor(getClass().getClassLoader()), liquibaseConnection);
 	}
 	
-	protected void initializeDatabase() throws SQLException, ClassNotFoundException {
-		String driver = "org.h2.Driver";
-		Class.forName(driver);
+	protected void initializeDatabase() throws ClassNotFoundException {
+		if(useInMemoryDatabase()) {
+			String driver = "org.h2.Driver";
+			Class.forName(driver);
+		} else {
+			setupContainerDB();
+		}
 	}
-	
+
+	private void setupContainerDB() throws ClassNotFoundException {
+		Containers.ensureDatabaseRunning();
+		Class.forName(Containers.getDatabaseDriverClassName());
+		CONNECTION_URL = Containers.getDatabaseURL();
+	}
+
 	protected void updateDatabase(String filename) throws Exception {
 		Liquibase liquibase = getLiquibase(filename);
 		liquibase.update(new Contexts(CONTEXT));
 		liquibase.getDatabase().getConnection().commit();
 	}
-	
+
 	protected void dropAllDatabaseObjects() throws SQLException {
-		Connection connection = getConnection();
-		Statement statement = null;
```

Comment:
Thinking about this again, was the method meant to also deal with MySQL and PostgreSQL?

Reply:
This method is used to restart the database. So for MySQL, I think deleting and creating a new database works.